gm4_disassemblers/data/gm4_disassemblers/functions/check_recipe.mcfunction

```diff
@@ -0,0 +1,10 @@
+# processes each item
+# @s = armor stand display (we need the hand) [tag=gm4_disassembler_stand]
+# located at the disassembler block
+# run from extract
+data modify entity @s HandItems[0] set from storage gm4_disassemblers:temp Items[0]
+execute store result score $damage gm4_disassembler run data get storage gm4_disassemblers:temp Items[0].tag.Damage
+scoreboard players set dropped gm4_disassembler 0
+function #gm4_disassemblers:before_base
+execute if score dropped gm4_disassembler matches 0 run function #gm4_disassemblers:during_base
+execute if score dropped gm4_disassembler matches 0 run function #gm4_disassemblers:after_base
```

Comment:
Why did you rename this file to the singular form? This isn't in line with the usual naming convention for custom-crafting type modules.

Reply:
Honestly, I deleted all the files and just created new ones from scratch. Really just a typo.